app/uk/gov/hmrc/economiccrimelevyaccount/utils/Constants.scala

```diff
@@ -16,14 +16,20 @@
 
 package uk.gov.hmrc.economiccrimelevyaccount.utils
 
-object Constants {
+trait Constants
 
-  val TRM_NEW_CHARGE           = "TRM New Charge"
-  val TRM_AMEND_CHARGE         = "TRM Amend Charge"
-  val INTEREST_DOCUMENT        = "Interest Document"
-  val PAYMENT                  = "Payment"
-  val CONTRACT_OBJECT_TYPE_ECL = "ECL"
-  val SESSION_ID_HEADER_NAME   = "x-session-id"
-  val HEADER_X_CORRELATION_ID  = "x-correlation-id"
+object DocumentTypeValues extends Constants {
```

Comment:
Normally enums and psuedo-enum constants use a singular name instead of plural.

**DocumentType.TrmNewCharge** reads better than **DocumentTypeValues.TrmNewCharge** for example

The suffix adds now value and increases cognitive load (ever so slightly) when reading the code.